policy/modules/kernel/kernel.if

```diff
@@ -1076,6 +1072,24 @@ interface(`kernel_mounton_proc_dirs',`
 	allow $1 proc_t:dir mounton;
 ')
 
+########################################
+## <summary>
+##	Mount on the files in /proc.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`kernel_mounton_proc_files',`
```

Comment:
Thie mounts in this file may fit into the `init_mountpoint`-style mentioned above.